training/src/main/java/com/wolox/training/models/Book.java

```diff
@@ -1,28 +1,26 @@
 package com.wolox.training.models;
 
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import java.util.ArrayList;
 import java.util.List;
 import javax.persistence.Column;
 import javax.persistence.Entity;
 import javax.persistence.GeneratedValue;
 import javax.persistence.GenerationType;
 import javax.persistence.Id;
 import javax.persistence.ManyToMany;
-import javax.persistence.SequenceGenerator;
 import lombok.AccessLevel;
 import lombok.Getter;
-import lombok.NoArgsConstructor;
 import lombok.Setter;
 
 @Entity
 @Getter
 @Setter
-@NoArgsConstructor
 public class Book {
 
   @Id
   @Setter(AccessLevel.NONE)
-  @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "BOOK_SEQ")
-  @SequenceGenerator(name = "BOOK_SEQ", sequenceName = "BOOK_SEQ")
+  @GeneratedValue(strategy = GenerationType.IDENTITY)
```

Comment:
I think it's not necessary to change the GeneratedValue, it should work just fine. The problem with identity is that if you insert the first book it will be inserted with id 1. Then, when you insert the first user, it will be inserted with id 2. Because this autoincrement for postgresql is shared between all tables. If you have any problem with that, let me know and maybe I can help you :)